### PR TITLE
flannel: update docker_opts path

### PR DESCRIPTION
**Pull request by crawford**
From https://github.com/coreos/bugs/issues/1756.

### CODE REVIEW

**Comment by robszumski** [comment]
@crawford Do you think this doc should be backported?

**Comment by crawford** [comment]
Backported to what?

**Comment by robszumski** [comment]
To here: https://coreos.com/flannel/docs/latest/flannel-config.html instead of waiting for the next flannel release.

**Comment by crawford** [comment]
Oh, sure. I wouldn't have called that "backporting". You threw me off.

File: flannel/flannel-config.md [hunk 1]
```diff
@@ -157,5 +157,5 @@ Here is the sequence of events that happens when `flanneld.service` is started f
 2. flanneld starts and writes out `/run/flannel/subnet.env` with the acquired IP subnet information.
 3. `ExecStartPost` in `flanneld.service` converts information in `/run/flannel/subnet.env` into Docker daemon command line args (such as `--bip` and `--mtu`), storing them in `/run/flannel_docker_opts.env`.
```
Comment (robszumski) [inline flannel/flannel-config.md]: Does this need to be changed here as well?

Comment (crawford) [inline flannel/flannel-config.md]: shit

**Pull request merged**